src/webextension/widgets/button.css

```diff
@@ -27,11 +27,6 @@
   background-color: #003eaa;
 }
 
-.normal-theme:hover,
-.ghost-theme:hover {
-  background-color: rgba(12, 12, 13, 0.2);
-}
-
```

Comment:
These lines were duplicated on L39-L42 below.

Reply:
No they weren't. The lines below need `:active` appended to them, which I'm doing in another PR. I'm good at copy-paste. :/